src/sentry/integrations/slack/utils/notifications.py

```diff
@@ -65,11 +66,17 @@ def _get_thread_config(
     parent_notification_message: (
         NotificationActionNotificationMessage | MetricAlertNotificationMessage | None
     ),
+    incident_status: IncidentStatus,
 ) -> tuple[bool, str | None]:
     if parent_notification_message is None:
         return False, None
 
-    return True, parent_notification_message.message_identifier
+    reply_broadcast = False
```

Comment:
nit: `reply_broadcast` is a little confusing of a variable name for me. maybe `is_thread_reply`?

we also don't have to change it here -- i noticed this variable has been around for a bit, so maybe it's just something to think about for notification plat.